app/code/Magento/Analytics/ReportXml/Query.php

```diff
@@ -82,6 +82,7 @@ public function getConfig()
      * @return mixed data which can be serialized by <b>json_encode</b>,
      * which is a value of any type other than a resource.
      */
+    #[\ReturnTypeWillChange]
```

Comment:
I would recommend to specify a proper return type for this method instead

Reply:
Hi @fascinosum , in PHP >= 8 the return type for this method is _mixed_  (https://www.php.net/manual/en/jsonserializable.jsonserialize.php) , but in PHP 7.4 this return type is not specified yet.